### DISCUSSION TITLE
Undefined variable: component on tools/filters views

### DISCUSSION THREAD

**Discussion by lutfifirdaus** [rappasoft/laravel-livewire-tables#695] [Q&A] [answered]
Hello everyone🙌
Today i got an error when updating this rappasoft/laravel-livewire-tables from v.1 to v.2 on my project. i've been following the upgrade guideline but after all, i've got error like this

`Undefined variable: component (View: C:\xampp\htdocs\ppdb-ydai\resources\views\vendor\livewire-tables\components\tools\sorting-pills.blade.php)`

Can anyone help me to find solution from this error?

**Comment by rappasoft** [comment, marked as answer]
Make sure you republish all views if you published for v1.

**Reply by lutfifirdaus** [reply]
hello @rappasoft, thanks for the answer. I've been solve this error, the laravel version that i'm using is 8.61 is i assume it doesn't work with `@aware(['component'])` , so when i upgrade it to version 8.83.7 its work.
